src/overlay/Peer.h

```diff
@@ -98,10 +100,10 @@ class Peer : public std::enable_shared_from_this<Peer>,
         uint64_t mAsyncWrite;
         uint64_t mMessageDrop;
 
-        std::chrono::milliseconds mMaxMessageDelayInWriteQueue;
-        std::chrono::milliseconds mMaxMessageDelayInAsyncWrite;
-        std::chrono::milliseconds mMaxOutboundQueueDelaySCP;
-        std::chrono::milliseconds mMaxOutboundQueueDelayTxs;
+        medida::Timer& mMessageDelayInWriteQueueTimer;
```

Comment:
We shouldn't duplicate these metrics. You can get the data you need in `getJsonInfo` by querying the app-wide metric registry. See `Application::getMetrics`

Reply:
actually, scratch that, we don't track per-peer metrics in the global metric registry. So I think we just need to track these timers in the struct itself (we don't need a registry for this)

Reply:
Updated it!